packages/core/src/modules/dids/methods/sov/IndySdkSovDidRegistrar.ts

```diff
@@ -170,6 +167,71 @@ export class SovDidRegistrar implements DidRegistrar {
       },
     }
   }
+
+  public async registerPublicDid(
+    agentContext: AgentContext,
+    submitterDid: string,
+    targetDid: string,
+    verkey: string,
+    alias: string,
+    role?: Indy.NymRole
+  ) {
+    const pool = this.indyPoolService.ledgerWritePool
```

Comment:
This takes the pool from the ledgerWritePool, however you also added a config option to set the indyNamespace. I think we should get the pool based on the indyNamespace from the indyPoolService and use that as the pool to write to the ledger instead.